ext/fuzztest.cmd

```diff
@@ -9,10 +9,12 @@
 
 git clone https://github.com/google/fuzztest.git
 cd fuzztest
-: # There is no tagged release as of 2024/02/23. Pick the last commit that works.
-git checkout b39227cb001a46ed007fa37e40507d777652ede9
+: # There is no tagged release as of 2024/04/09. Pick the latest commit that works.
+git checkout a990537097b6fd337436f7662fb39fc91fa01f3f
 sed -i 's/-fsanitize=address//g' ./cmake/FuzzTestFlagSetup.cmake
 sed -i 's/-DADDRESS_SANITIZER//g' ./cmake/FuzzTestFlagSetup.cmake
+sed -i 's/if (IsEnginePlaceholderInput(data)) return;/if (data.size() == 0) return;/g' ./fuzztest/internal/compatibility_mode.cc
+sed -i 's/set(GTEST_HAS_ABSL ON)/set(GTEST_HAS_ABSL OFF)/g' ./cmake/BuildDependencies.cmake
```

Comment:
It would be good to add a comment to explain why we need to patch cmake/BuildDependencies.cmake. (The reason was not given in the commit message.) Thanks!

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2096